src/DGtal/io/viewers/Viewer3D.ih

```diff
@@ -38,6 +38,11 @@
 #include "DGtal/io/viewers/windows/GL/glext.h"
 #endif
 
+//On macos, disabling GL deprecated
+#ifndef __linux__
```

Comment:
And what about other OS (e.g. Windows)?

Reply:
I've removed it.. Not working even on Mac ;)